.github/workflows/run-forge.yaml

```diff
@@ -130,32 +112,17 @@ jobs:
 
           # source this script to run and get environment variables
           source testsuite/run_forge.sh
-
-          # append some GHA-specific content to the comment
```

Comment:
are these comment additions now done in the forge wrapper?

Reply:
Yeah these are all done in the forge wrapper, source testsuite/run_forge actually exec the forge.py so this never runs anymore